plasTeX/Imagers/__init__.py

```diff
@@ -638,12 +638,18 @@ def compileLatex(self, source):
                         cmd_line))
         output = None
         for ext in ['.dvi','.pdf','.ps']:
-            if os.path.isfile('images'+ext):
-                output = open('images'+ext, 'rb')
+            try:
+                with open('images'+ext, 'rb') as f:
+                    output = open('images'+ext, 'rb').read()
                 break
+            except OSError:
+                # Ignore if file not found
+                pass
 
         # Change back to original working directory
         os.chdir(cwd)
+        if not self.config["images"]["save-file"]:
```

Comment:
Now I'm confused. I was about to write this should be added in https://github.com/plastex/plastex/blob/master/plasTeX/Config.py but I see it's already there, added 14 years ago in https://github.com/plastex/plastex/commit/d9c53162a3252d0d5651f0d3fe16d4ef76dc5ab0. Did you see it used anywhere? It would be nice to dig a bit into the git history of the project to see what happened here. In any case, did you make sure this option is mentioned in the docs?